fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/PillButtonTokens.kt

```diff
@@ -0,0 +1,331 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.ThemeMode
+import com.microsoft.fluentui.theme.token.*
+import kotlinx.parcelize.Parcelize
+
+enum class PillButtonStyle {
```

Comment:
Put PillButtonStyle to common place in core as the many control need to have these 2 variant.

Reply:
Moved to Core/FluentStyle